.github/workflows/regression_tests.yml

```diff
@@ -76,22 +76,14 @@ jobs:
       - name: Install ECWAM dependencies
         run: |
           sudo apt-get -o Acquire::Retries=3 install -y libopenmpi-dev
-          source loki-activate
-          pip install fypp
+          pip install pyyaml fypp
 
       - name: Run CLOUDSC and ECWAM regression tests
         env:
           CLOUDSC_DIR: ${{ github.workspace }}/cloudsc
           CLOUDSC_ARCH: ${{ github.workspace }}/cloudsc/arch/github/ubuntu/gnu/9.4.0
           ECWAM_DIR: ${{ github.workspace }}/ecwam
-          ECWAM_ARCH: ${{ github.workspace }}/ecwam/arch/github/ubuntu/gnu/9.4.0
```

Comment:
Just double-checking: Is relying on default arch safe here? @awnawab

Reply:
My understanding is that implies no arch files are used, i.e., no specific toolchain.cmake and env.sh. Since none are required, this should be fine.

Reply:
Yes exactly, the ecwam CI also runs without any arch files and just relies on the default cmake/ecbuild flags plus those set by `ecwam_compile_flags.cmake`.